discovery-handlers/onvif/src/discovery_handler.rs

```diff
@@ -583,4 +586,64 @@ mod tests {
             .await
             .is_none());
     }
+
+    #[tokio::test]
+    async fn test_apply_filters_include_mac_exist_different_cases() {
```

Comment:
nit (and for other test)
```suggestion
    async fn test_apply_filters_include_mac_exist_different_letter_cases() {
```

Reply:
rename function